mozilla-release/toolkit/mozapps/installer/linux/rpm/mozilla.spec

```diff
@@ -20,11 +20,12 @@ BuildRoot:      %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
 
 #AutoProv:       no
 
-BuildRequires:  desktop-file-utils
+# CLIQZ: on ubuntu dependency has to be installed with apt-get
+#BuildRequires:  desktop-file-utils
```

Comment:
I'd suggest wrapping it inside a deb-specific condition:
```
%if %{?alien:1}
%else
BuildRequires:  desktop-file-utils
%endif
```

and then set `alien` variable when building an `rpm` only if it's intended to be converted to `deb`:
```
RPM_CMD += \
  --define 'alien yes'
```

You can find examples of specifying conditions in the `.spec` file as well as in `upload-files.mk`.